internal/website/data/examples.json

```diff
@@ -127,22 +127,6 @@
 		"imports": "import (\n\t\"context\"\n\n\t\"gocloud.dev/pubsub\"\n\t_ \"gocloud.dev/pubsub/gcppubsub\"\n)",
 		"code": "topic, err := pubsub.OpenTopic(ctx, \"gcppubsub://myproject/mytopic\")\nif err != nil {\n\treturn err\n}\ndefer topic.Shutdown(ctx)"
 	},
-	"gocloud.dev/pubsub/kafkapubsub.ExampleOpenSubscription": {
```

Comment:
This change to `examples.json` looks wrong. We probably need to run `gatherexamples` over all modules or something...?

Reply:
Yep, fixed the gathering script to use allmodules